src/CreeDictionary/phrase_translate/crk_tag_map.py

```diff
@@ -13,6 +13,8 @@
     ("+Obv", COPY_TAG_NAME, 1),
     ("+Loc", COPY_TAG_NAME, 1),
     ("+Distr", COPY_TAG_NAME, 1),
+    ## XXX FIXME I have no idea what +Voc is but it causes problems so drop it
```

Comment:
`Voc` = [vocative](https://en.wikipedia.org/wiki/Vocative_case). It's the form of the noun you use when speaking to someone. Some languages have a special form of the noun for this.